github-profiles/rrodriguezca-2238/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8"/> 
+    <h1>Robinson Rodríguez CV</h1>
+</head>
+<style>
+    body {
+  margin: 0 auto;
+  max-width: 50em;
+}
+    body {
+  font-family: "Helvetica", "Arial", sans-serif;
+}
+body {
+  line-height: 1.5;
+  padding: 4em 1em;
+}
+
+h2 {
+  margin-top: 1em;
+  padding-top: 1em;
+}
+body {
+  color: #555;
+}
+pre {
+  background: #eee;
+}
+body {
+  color: #566b78;
+}
+h1 {
+  background-color: #5a8891;
+  
+  background-position: center top;
+  background-repeat: no-repeat;
+  background-size: cover;
+  line-height: 1.2;
+  padding: 10vw 2em;
+  text-align: center;
+}
+h1 {
+  color: white;
+  font-size: 2.5em;
+  font-weight: 300;
+}
+h1 {
+  border: 1px solid #e81c4f;
+  border-radius: 290486px;
+  color: white;
+  font-size: 3em;
+  letter-spacing: 0.2em;
+  padding: 1em 2em;
+  text-transform: uppercase;
+  text-decoration: none;
+  transition: none 200ms ease-out;
+  transition-property: color, background;
+}
+</style>
+<body>
+        <h2>Description</h2>
+
+    <p>I am teacher of English on a quest to learn something new. This new knowledge would be integrated as part of an ongoing effort to become a full stack developer. </p>
```

Comment:
I suggest using Prettier to format the document and fix any strange-looking indentations like you have here in line 64
```suggestion
    <p>I am teacher of English on a quest to learn something new. This new knowledge would be integrated as part of an ongoing effort to become a full stack developer. </p>
```

Reply:
I did not use Prettier, but I coded  much more organized tables. does it work?